applications/FluidDynamicsApplication/python_scripts/apply_mass_conservation_check_process.py

```diff
@@ -1,17 +1,21 @@
+from __future__ import absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
 import KratosMultiphysics
 import KratosMultiphysics.FluidDynamicsApplication as KratosFluid
 
 def Factory(settings, Model):
     if( not isinstance(settings, KratosMultiphysics.Parameters) ):
         raise Exception("expected input shall be a Parameters object, encapsulating a json string")
-    return ApplyMassConservationCheckProcess(Model, settings["Parameters"])
+    return ApplyMassConservationCheckProcess( Model, settings["Parameters"] )
 
 class ApplyMassConservationCheckProcess(KratosMultiphysics.Process):
 
     def __init__(self, Model, settings):
 
         KratosMultiphysics.Process.__init__(self)
 
+        self._my_log_file = "Mass_Conservation.log"
```

Comment:
I would put the file name as a parameter of the default JSON settings below so a user-defined file name can be specified if needed (e.g. imagine that you want to compare the mass conservation of multiple runs).

Reply:
Changed to a parameter in the *.json file.